### PR TITLE
Gnome 45 support

### PR DESCRIPTION
**Pull request by axxapy**
_Thanks for an awesome extension!_

This is my "to go" extension so I had to port it to gnome 45...

I followed https://gjs.guide/extensions/upgrading/gnome-shell-45.html

One note: due to major changes in gnome 45 it is not possible to support older versions. It's either <=44 or 45.
extensions.gnome.org supports having multiple versions => so it should be fine for users who're still on <45

### CODE REVIEW

**Comment by wbolster** [comment]
wow, this looks fantastic. i'll give it a try this week, and will get back to you if i encounter any problems.

**Comment by wbolster** [comment]
i gave this a try the past few days and it works flawlessly. thanks a lot! 🥳

**Comment by wbolster** [comment]
fyi, i've uploaded a new version to https://extensions.gnome.org/

hopefully available soon (after review)

**Comment by TheWiseNoob** [comment]
> fyi, i've uploaded a new version to https://extensions.gnome.org/
> 
> hopefully available soon (after review)

@wbolster I'm curious. How long do they usually take to approve an extension?

Thanks for your work on this! I use this extension daily for work and gaming.

**Comment by wbolster** [comment]
> How long do they usually take to approve an extension?

my experience is a few days max; but since this is a major rewrite (of the UI components) it may take a bit longer.

File: prefs.js [hunk 1]
```diff
@@ -1,148 +1,130 @@
...
-}
-
-function is_playing_sound_supported() {
-  try {
-    imports.gi.Gst;
-    imports.gi.GstAudio;
-    return true;
-  } catch(e) {
-    log(`${Extension.metadata.uuid}: Playing sound is not supported on this system. Is GStreamer package installed?`);
-    log(`${Extension.metadata.uuid}: ${e}`);
```
Comment (szpak) [inline prefs.js:144]: Slightly late for the review, but @axxapy I wonder, if any warning is printed in the logs or only the message displayed in preferences are shown?

**Pull request merged**